src/components/complex/game/index.tsx

```diff
@@ -1,10 +1,10 @@
-import React from "react"
-import { initGame } from "./game"
+import React from 'react'
+import { initGame } from './game'
 
 export type GameType = React.FC
 
 export const Game: GameType = () => {
-  return <canvas ref={(elem: HTMLCanvasElement) => initGame(elem)}></canvas>
+	return <canvas ref={(elem: HTMLCanvasElement) => initGame(elem)}></canvas>
```

Comment:
```suggestion
	return <canvas ref={(elem: HTMLCanvasElement) => elem && initGame(elem)}></canvas>
```

Should probably fix the undefined canvas error.

Reply:
That fixed the undefined canvas but it's still creating another game instance. Hopefully something can be done about that later without much trouble.